lib/simp/cli/utils.rb

```diff
@@ -65,6 +72,38 @@ def get_config(environment='production', section='master')
 
       return %x{puppet config print --environment=#{environment} --section=#{section}}.lines
     end
+
+    # Add another quick shortcut if possible
+    def is_pe?
+      return (
+        (@system_puppet_info && @system_puppet_info[:puppet_group] == 'pe-puppet') ||
+        Simp::Cli::Utils.is_pe?
+      )
+    end
+  end
+
+  # Try to determine if we are on a PE server in as many ways as possible
+  #
+  # @return [Boolean]
+  #   `true` if PE detected, `false` otherwise
+  def self.is_pe?
+    require 'facter'
+
+    # From cheapest to most expensive
+    return Facter.value('is_pe') if Facter.value('is_pe')
```

Comment:
In the future, would be nice to have an acceptance test with PE loaded to see when all these guesses fail.  Would be our early warning system because of Puppet changes  `simp config` and `simp bootstrap` would not work in PE.

Reply:
Will be adding an acceptance test to `simp-core` after all the current work gets in so that I know I have something that works to start with.